estate/models/__init__.py

```diff
@@ -0,0 +1,6 @@
+from . import (
+    estate_property,
+    estate_property_type,
+    estate_property_tag,
+    estate_property_offer,
+)
```

Comment:
```suggestion
from . import estate_property
from . import estate_property_type
from . import estate_property_tag
from . import estate_property_offer
```
We usually do the imports in the `__init__` like this if you look into Odoo.